File: LifeInsuranceAPI/Models/UserDetails.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace LifeInsuranceAPI.Models
{
    public class UserDetails
    {
        [Key]
        public int Id { get; set; }
        public string DateOfBirth { get; set; }

        public string MartialStatus { get; set; }

        public string Occupation { get; set; }

        
        public string Salary { get; set; }

        
        
        public string AadharNumber { get; set; }

      
        public string PanNumber { get; set; }


        
        public string StreetAddressLine1 { get; set; }

        
        public string StreetAddressLine2 { get; set; }

       
        public string City { get; set; }

        
        public string State { get; set; }

        
        public string ZipCode { get; set; }

        public int TenureOfPolicy { get; set; }


        public User User { get; set; }
        public int UserId { get; set; }
        
        public Policy Policy { get; set; }
        public int PolicyId { get; set; }
    }
}

File: LifeInsuranceUI/src/app/models/user-details.ts
import { Policy } from "./policy";
import { User } from "./user";
export class UserDetails {
    Id: number;
    DateOfBirth: string;
    MartialStatus: string;
    Occupation: string;
    Salary: string;
    AadharNumber: string;
    PanNumber: string;
    StreetAddressLine1: string;
    StreetAddressLine2: string;
    City: string;
    State: string;
    ZipCode: string;
    TenureOfPolicy: number;
    Policy: Policy;
    PolicyId: number;
    User: User;
    UserId: number;
}
